src/main/java/school/lemon/changerequest/java/introduction/hw1/Task5.java

```diff
@@ -2,11 +2,22 @@
 
 
 public class Task5 {
+
+
+
+
+
+
     public static double calculateA(double x, double y, double z) {
-        return 0;
+
+        return (2*(Math.sin(x-(Math.PI) / 6))*calculateB(z)) / (0.5 + Math.pow(Math.sin(y), 2));
     }
 
     private static double calculateB(double z) {
-        return 0;
+
+        return (1 + Math.pow(z, 2) / (3 + (Math.pow(z, 2)) / 5));
```

Comment:
it's good practice to save results of some calculations, so that your program don't need to calculate same things many times.
In following case, you can save result of `Math.pow(z,2)` into some handy variable and reuse it wherever you need.